### ISSUE TITLE
out[87]: tech 请问这tech结果的输入是？code里面，在哪输入待分类语句呢？我有新的句子怎么输入？谢谢

### ISSUE THREAD

**Issue by tectal** [GhibliField/Multiclass-Text-Classification-using-BERT#2]
In [87]:
logits=model(input_ids, segment_ids, input_mask)
sigmoid = nn.Sigmoid()
probs = sigmoid(logits)
predicted_labels = probs.detach().cpu().numpy() > 0.5
list(label2idx.keys())[list(label2idx.values()).index(np.argwhere(predicted_labels[0] ==True).item())]

out[87]: tech 请问这tech结果的输入是？code里面，在哪输入待分类语句呢？我有新的句子怎么输入？谢谢

**Comment by tectal** [comment]
@GhibliField 您好？

**Comment by GhibliField** [comment]
> In [87]:
> logits=model(input_ids, segment_ids, input_mask)
> sigmoid = nn.Sigmoid()
> probs = sigmoid(logits)
> predicted_labels = probs.detach().cpu().numpy() > 0.5
> list(label2idx.keys())[list(label2idx.values()).index(np.argwhere(predicted_labels[0] ==True).item())]
> 
> out[87]: tech 请问这tech结果的输入是？code里面，在哪输入待分类语句呢？我有新的句子怎么输入？谢谢

Thanks for your question! I've added some unseen texts as examples, you may have a check on the last cell of the notebook.
